### PR TITLE
Admin profile layout added 

### PR DESCRIPTION
**Pull request by Nimit3-droid**
Admin profile added with Display all currently issued items , Access Managing and Role Assigning, Will be able to see all the pending requests and history of transitions(incomplete)

### CODE REVIEW

**Comment by Anshultilondiya** [comment]
Your branch has conflicts in App.css file with upstream

**Comment by Nimit3-droid** [comment]
> Your branch has conflicts in App.css file with upstream

@Anshultilondiya , I have resolved the commits.

File: frontend/src/components/profiles/AdminProfile.js [hunk 1]
```diff
@@ -1,13 +1,88 @@
-import React from 'react';
+import React from 'react'
+import Item from './Item'
+import { Avatar,Box,Button,TextField } from '@material-ui/core'
+import '../../App.css'
```
Comment (Anshultilondiya) [inline frontend/src/components/profiles/AdminProfile.js:4]: unnecessary import

App.css affects globally, we don't have to import it anywhere other than App.js

File: frontend/src/components/profiles/Item.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Box ,Button} from '@material-ui/core'
+import '../../App.css'
```
Comment (Anshultilondiya) [inline frontend/src/components/profiles/Item.js:3]: remove this import

**Review by Anshultilondiya — CHANGES_REQUESTED** [review]
Do the needful

File: frontend/src/components/profiles/Item.js (hunk 2 above)
Comment (Nimit3-droid) [reply frontend/src/components/profiles/Item.js]: removed imports

**Review by Anshultilondiya — APPROVED** [review]

**Pull request merged**